tasks/build.yml

```diff
@@ -62,12 +64,12 @@
   service:
     name: go-server
     enabled: yes
-    state: stopped
 
 - name: Install go-server plugins
   become: yes
-  become_user: "{{ gocd_server.user }}"
+  become_user: "{{ sansible_gocd_server_user }}"
   get_url:
-    dest: /home/{{ gocd_server.user }}/work/go-server/plugins/external
-    url:  "{{ item }}"
-  with_items: "{{ gocd_server.plugins }}"
+    dest: "/home/{{ sansible_gocd_server_user }}/work/go-server/plugins/external"
+    url: "{{ item }}"
+  with_items: "{{ sansible_gocd_server_plugins }}"
+  notify: restart go-server
```

Comment:
We normally use list format for notifies:

```YAML
notify:
  - restart go-server
```

Reply:
I have _no_ opinion on this :-)